hack/tools/kind-config.yaml

```diff
@@ -9,4 +9,4 @@ nodes:
 containerdConfigPatches:
 - |-
   [plugins."io.containerd.grpc.v1.cri".registry.mirrors."localhost:5000"]
-    endpoint = ["https://kind-registry:5000"]
\ No newline at end of file
+    endpoint = ["http://kind-registry:5000"]
```

Comment:
I am not sure I understand. So the kind-registry is still created but localhost:5000 is pointing to kind-registry:5000?